lntest/itest/log_error_whitelist.txt

```diff
@@ -140,6 +140,10 @@
 <time> [ERR] LNWL: Notifying unmined tx notification (<hex>) while creating notification for blocks
 <time> [ERR] LNWL: Rescan for <num> addresses failed: the client has been shutdown
 <time> [ERR] LTND: Unable to lookup witness: no witnesses
+<time> [ERR] LTND: Error closing channel state database: context canceled
```

Comment:
nit: I think we can filter out the context cancelled error messages when we call the close funcs.